pkg/server/server.go

```diff
@@ -1297,3 +1337,12 @@ func (s *Server) resolveTypesystem(ctx context.Context, storeID, modelID string)
 
 	return typesys, nil
 }
+
+// If the requested consistency preference is HIGHER_CONSISTENCY, but the experimental flag is not enabled,
+// returns an error.
+func (s *Server) validateConsistencyRequest(c openfgav1.ConsistencyPreference) error {
+	if !s.IsExperimentallyEnabled(ExperimentalEnableConsistencyParams) && openfgav1.ConsistencyPreference_HIGHER_CONSISTENCY == c {
```

Comment:
**Question:** Should we not also error when the `MINIMIZE_LATENCY` option is passed too? If the experimental flag isn't enabled, it may give the incorrect impression that their configuration was successfully applied; it may not be apparent that it's the default behavior by happenstance.

Reply:
Yup this should most likely be checking it it's not `UNSPECIFIED`.